veevavault/1.3/modules/ROOT/pages/veevavault-connector-config-topics.adoc

```diff
@@ -806,24 +222,24 @@ The following are the error codes that are caught in the Error Handling componen
 [[streaming_and_pagination]]
 == Streaming and Pagination
 
-All the operations in the connector (except Download Document) return an InputStream as a payload with respective results based on operation output. Because of this, by default, Mule applies Streaming Strategies. See xref:mule-runtime::streaming-about.adoc[Mule Streaming Strategies] for more details. The streaming strategies configuration fields are in the *Advanced* tab of the connector operations.
+All the operations in the connector (except *Download Document*) return an InputStream as a payload with respective results based on operation output. Because of this, by default, Mule applies Streaming Strategies. See xref:mule-runtime::streaming-about.adoc[Mule Streaming Strategies] for more details. The streaming strategies configuration fields are in the *Advanced* tab of the connector operations.
```

Comment:
```suggestion
With the exception of *Download Document*, all connector operations return an input stream as a payload, with respective results based on operation output. Because of this, by default Mule applies streaming strategies. See xref:mule-runtime::streaming-about.adoc[Mule Streaming Strategies] for more details. The streaming strategies configuration fields are in the *Advanced* tab of the connector operations.
```